eks-distro-base/scripts/clean_install

```diff
@@ -35,17 +35,17 @@ fi
 
 RPMS=(${PACKAGES// / })
 for rpm in "${RPMS[@]}"; do
-    # if installed already skip
-    if rpm --root $NEWROOT -q --quiet $rpm ; then
-        continue
-    fi
-
     if [ ! $SHALLOW ]; then
         yum --installroot $NEWROOT install -y $rpm
     else
         yumdownloader --destdir=$DOWNLOAD_DIR -x "*.i686" $rpm
         rpm_file=$(ls $DOWNLOAD_DIR/$rpm-*.rpm)
-        rpm -ivh --nodeps --root $NEWROOT $JUSTDB $rpm_file
+        
+        # if installed already skip
+        if ! rpm --root $NEWROOT -q --quiet $rpm ; then
```

Comment:
This will allow us to install bins from packages that have already been "installed" in subsequent image builds, like on the csi-driver build itself.